### DISCUSSION TITLE
Best compatible linux distro

### DISCUSSION THREAD

**Discussion by lblum-steno** [opensteno/plover#1683] [Q&A and Support]
Hi there,

I have problem with Plover on my Linux distro - Ubuntur LTS 24.04 (see bug section)...which 'forces' me to go to Windows to use plover..
Do you have experience with LInux distro where Plover works correctly? Or features needed so that it works correctly such as Debian vs .. and so on?

Thank you very much!

Lionel
(a still learning steno hobbyis)

**Comment by aerickt** [comment]
Development for Plover has unfortunately stalled as there aren't any active maintainers at the moment. One shortcoming of Plover because of this is that it doesn't run on distros that use Wayland (which Ubuntu does by default). In fact, most mainstream Linux distros are switching to Wayland. That being said, there are some workarounds: https://plover.wiki/index.php/Wayland

Beyond this, however, there aren't really any compatibility issues between different Linux distros. Plover should work on any distro that supports AppImages (though, even the AppImage itself can have dependency issues). You may find issues like words being jumbled due to Plover's fast emulated output, but this is fixed pretty easily by setting a key delay in Plover's settings.

**Reply by lblum-steno** [reply]
Thanks I finally managed to make it run under Ubuntu 24.04 LTS.
I however do not see any plugins appearing under plugins manager (I use Lapwing). Any idea? 

Many thanks!
